### PR TITLE
Document highlights on yield keywords highlight other occurrences in …

### PR DESCRIPTION
**Pull request by Zzzen**
…the same body

See https://github.com/Microsoft/TypeScript/pull/23923#issuecomment-387804168

### CODE REVIEW

File: tests/cases/fourslash/getOccurrencesYield.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+////  }
+////  return function* g() {
+////    yield 1;
+////  }
+////}
+
+
+verify.rangesAreOccurrences(false);
+
+goTo.marker();
```
Comment (andy-ms) [inline tests/cases/fourslash/getOccurrencesYield.ts]: Don't think the rest of the test is necessary -- `verify.rangesAreOccurrences` should already loop over all ranges and test that all the other ranges are document highlights of it. See `verifyRangesAreOccurrences` in `fourslash.ts`.

**Comment by mhegazy** [comment]
@Zzzen can you merge master into your branch again to get the tests to pass.

File: tests/cases/fourslash/getOccurrencesYield.ts (hunk 1 above)
Comment (Zzzen) [reply tests/cases/fourslash/getOccurrencesYield.ts]: I saw this pattern in many fourslash tests so I just copy & paste it. Maybe we should remove these redundant/misleading code.

Comment (andy-ms) [reply tests/cases/fourslash/getOccurrencesYield.ts]: :+1: go for it

**Pull request merged**